Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -210,19 +211,33 @@ void MenuBar::AddGameListTypeSection(QMenu* view_menu)
   connect(list_view, &QAction::triggered, this, &MenuBar::ShowList);
 }
 
-// TODO implement this
 void MenuBar::AddTableColumnsMenu(QMenu* view_menu)
 {
+  const static QMap<QString, bool*> columns{{tr("Platform"), &Settings().IsPlatformVisible()},
```

Comment:
The preferred (or rather dominant) formatting in the codebase orders these as `static const`, or more abstractly, the general form is `[lifetime specifier] [mutability specifier] [volatility specifier]`